server/src/sessionManagement.mjs

```diff
@@ -31,13 +32,16 @@ class FireboltCall {
     }
 }
 
-class Session{
+class Session {
     #sessionStart;
     #sessionEnd;
     
     constructor(){
         this.calls = [];
         this.#sessionStart = Date.now();
+        this.sessionOutput = "log";
+        this.sessionOutputPath = "./sessions";
+        this.mockOutputPath = `./mocks/${this.#sessionStart}`;
```

Comment:
1) relative to current directory or <repoRoot>/mocks?
2) add to gitignore if not doing #3 below
3) unify all output so it's all under one directory ('output'?)

Reply:
1. Believe it is current directory, it will be saved under mock-firebolt/server
2. Adding output to gitignore
3. Making this change